astropy/utils/data.py

```diff
@@ -62,6 +62,10 @@
 _dataurls_to_alias = {}
 
 
+NOT_OVERWRITING_MSG = ('File {} already exists. If you mean to replace it '
```

Comment:
I wonder if this is more flexible... goes with my other suggestion, where you use this on a list of files.
```suggestion
NOT_OVERWRITING_MSG = ('File(s) {} already exists. If you mean to replace it '
```

More importantly though, I am not sure if `data.py` is the right place for this. How about moving it to `utils/misc.py`?

Reply:
Would it be a good idea to place the template somewhere in `io`?

Reply:
Probably more appropriate there but I am not sure exactly which module... Maybe @mhvk or @taldcroft can advise here...

Reply:
If we don't want to add a `io/utils.py`, then it can safely live in `__init__.py`. Importing the error message as
``from astropy.io import NOT_OVERWRITING_MSG`` will not cause import loops inside the ``io`` package, so it should be safe to use everywhere.

Reply:
`io/__init__.py` does sound like a good location, but I'll wait a while before making the change in case there are more opinions.